src/tests/commentCount.test.js

```diff
@@ -0,0 +1,9 @@
+const getCommentCount = async (itemId) => {
+  const response = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/HrIKPRrYjrxS00NlIVCD/comments?item_id=${itemId}`,
+  );
+  const data = await response.json();
+  return data.length;
+};
+
+export default getCommentCount;
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you have a test for have tests for all edge cases. Kindly think at least about the scenario in which there are 0 items on the page (the counter function should return 0) as stated by the project requirements.